gbfs.md

```diff
@@ -276,6 +276,7 @@ Field Name | Required | Type | Defines
 \- `propulsion_type` | Yes | Enum | The primary propulsion type of the vehicle. <br /><br />Current valid values are:<br /><ul><li>`human` _(Pedal or foot propulsion)_</li><li>`electric_assist` _(Provides power only alongside human propulsion)_</li><li>`electric` _(Contains throttle mode with a battery-powered motor)_</li><li>`combustion` _(Contains throttle mode with a gas engine-powered motor)_</li></ul> This field was inspired by, but differs from the propulsion types field described in the [Open Mobility Foundation Mobility Data Specification](https://github.com/openmobilityfoundation/mobility-data-specification/blob/master/provider/README.md#propulsion-types).
 \- `max_range_meters` | Conditionally Required | Non-negative float | If the vehicle has a motor (as indicated by having a value other than `human` in the `propulsion_type` field), this field is required. This represents the furthest distance in meters that the vehicle can travel without recharging or refueling when it has the maximum amount of energy potential (for example, a full battery or full tank of gas).
 \- `name` | Optional | String | The public name of this vehicle type.
+\- `reservation_time_minutes` | Optional | Non-negative integer | Time in minutes the vehicle can be reserved before rental is started. If set to values >0 the vehicle can be reserved for the specified amount of time. The vehicle remains locked until a rental is started. During this time the vehicle can not be reserved or rented by other users. If the reservation time is elapsed and no rental has been started, the reservation is terminated automatically and the vehicle is available for other users. If set to 0 then the vehicle can not be reserved. Note that a given reservation_time might only be one prerequisite and that in addition only specific tariffs might allow a reservation.
```

Comment:
There might be a typo here for "reservation_time" in "Note that a given reservation_time" (should be "reservation_time_minutes").

Reply:
Thanks for pointing out. I corrected the typo.